packages/fxa-auth-server/test/local/senders/select_email_services.js

```diff
@@ -719,173 +653,6 @@ describe('selectEmailServices:', () => {
-      assert.equal(args.length, 1);
-      assert.deepEqual(args[0], {
-        timeout: 100,
-      });
-
-      assert.equal(sandbox.run.callCount, 1);
-
-      args = sandbox.run.args[0];
-      assert.equal(args.length, 2);
-      assert.equal(
-        args[0],
-        'new RegExp("^foo@example\\.com$").test("foo@example.com")'
-      );
-      assert.equal(typeof args[1], 'function');
-    });
-
-    describe('call sandbox result handler with match:', () => {
-      beforeEach(() => {
-        sandbox.run.args[0][1]({ result: 'true' });
-        return promise;
-      });
-
-      it('resolved', () => {
-        assert.deepEqual(result, [
-          {
-            emailAddresses: ['foo@example.com'],
-            mailer: emailService,
-            emailService: 'fxa-email-service',
-            emailSender: 'sendgrid',
-          },
-        ]);
-      });
-
-      it('did not fail', () => {
-        assert.equal(failed, undefined);
-      });
-    });
-
-    describe('call sandbox result handler with timeout:', () => {
-      beforeEach(() => {
-        sandbox.run.args[0][1]({ result: 'TimeoutError' });
-        return promise;
-      });
-
-      it('resolved', () => {
-        assert.deepEqual(result, [
-          {
-            emailAddresses: ['foo@example.com'],
-            mailer: mailer,
-            emailService: 'fxa-auth-server',
-            emailSender: 'ses',
-          },
-        ]);
-      });
-
-      it('did not fail', () => {
-        assert.equal(failed, undefined);
-      });
-    });
-  });
-});
-
-describe('call selectEmailServices with mocked safe-regex, regex-only match and redos regex:', () => {
-  const emailAddress = 'foo@example.com';
-
-  let log, redis, safeRegex, mailer, emailService, selectEmailServices;
-
-  before(() => {
-    log = mocks.mockLog();
-    redis = {
-      get: sinon.spy(() =>
-        Promise.resolve(
-          JSON.stringify({
-            sendgrid: { regex: '^((((.*)*)*)*)*@example\\.com$' },
-          })
-        )
-      ),
-    };
-    // eslint-disable-next-line prefer-arrow-callback
-    safeRegex = sinon.spy(function() {
-      return true;
-    });
-    mailer = { mailer: true };
-    emailService = { emailService: true };
-
-    selectEmailServices = proxyquire(
-      `${ROOT_DIR}/lib/senders/select_email_services`,
-      {
-        '../redis': () => redis,
-        'safe-regex': safeRegex,
-      }
-    )(log, config, mailer, emailService);
-  });
-
-  it('email address was treated as mismatch', async () => {
-    const result = await selectEmailServices({ email: emailAddress });
-    assert.deepEqual(result, [
-      {
-        mailer: mailer,
-        emailAddresses: [emailAddress],
-        emailService: 'fxa-auth-server',
-        emailSender: 'ses',
-      },
-    ]);
-
-    assert.equal(safeRegex.callCount, 1);
-    const args = safeRegex.args[0];
-    assert.equal(args.length, 1);
-    assert.equal(args[0], '^((((.*)*)*)*)*@example\\.com$');
-  });
-});
-
```

Comment:
Look at all the tests you don't need when your regex library doesn't have any such thing as an "unsafe regex"!